app/controllers/application_controller.rb

```diff
@@ -1,5 +1,21 @@
 class ApplicationController < ActionController::API
 
+  protected
+
   include RescueFromUnlessLocal
 
+  def current_user_uuid
+    @current_user_uuid ||= begin
+      if Rails.env.development? && ENV['STUBBED_USER_UUID']
+        ENV['STUBBED_USER_UUID']
+      else
+        OpenStax::Accounts::Sso.user_uuid(request)
+      end
+    end
+  end
+
+  def render_unauthorized_if_no_current_user
+    head :unauthorized if current_user_uuid.nil?
```

Comment:
we dont want an error binding returned on this?  just the headers?

Reply:
The swagger docs don't have an error binding for the 401 error (only some of the error responses do): https://github.com/openstax/highlights-api/blob/master/app/controllers/api/v0/swagger_responses.rb#L4-L6